Library/Formula/corectl.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class Corectl < Formula
+  desc "CoreOS over OS X made very simple"
+  homepage "https://github.com/TheNewNormal/corectl"
+  url "https://github.com/TheNewNormal/corectl/archive/v0.0.18.tar.gz"
+  sha256 "8646c18ec0414a8f6e6330bc6720d9f0c4356ae7168b18cbc371220fe7a3e087"
+  head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+  depends_on "gnu-sed" => :build
+  depends_on :macos => :yosemite
+
+  def install
+    ENV["PATH"] = ENV["PATH"].sub("#{Superenv.bin}:", "")
```

Comment:
Good question. i added that snippet yesterday after a bit of googling. without it i always get ...
```
=> make corectl VERSION=0.0.18
fatal: Not a git repository (or any of the parent directories): .git
godep go build -o corectl
# github.com/TheNewNormal/libxhyve
cannot load DWARF output from $WORK/github.com/TheNewNormal/libxhyve/_obj//_cgo_.o: decoding dwarf section info at offset 0x0: too short
# github.com/TheNewNormal/corectl/uuid2ip
cannot load DWARF output from $WORK/github.com/TheNewNormal/corectl/uuid2ip/_obj//_cgo_.o: decoding dwarf section info at offset 0x0: too short
# github.com/yeonsh/go-ps
cannot load DWARF output from $WORK/github.com/yeonsh/go-ps/_obj//_cgo_.o: decoding dwarf section info at offset 0x0: too short
godep: go exit status 2
make: *** [corectl] Error 1
```